src/mscorlib/shared/System/ParseNumbers.cs

```diff
@@ -28,19 +28,14 @@ internal static class ParseNumbers
         private const int MinRadix = 2;
         private const int MaxRadix = 36;
 
-        public static unsafe long StringToLong(string s, int radix, int flags)
+        public static unsafe long StringToLong(ReadOnlySpan<char> s, int radix, int flags)
```

Comment:
Question: Is there a reason for this method to be unsafe? I might be missing something.

Reply:
Nope. But as I didn't add it and CI is green with signoffs, I won't remove it now.